scripts/prepare_sector_network.py

```diff
@@ -964,8 +993,8 @@ def add_shipping(n, costs):
         co2 = (
             shipping_oil_share
             * ports["p_set"].sum()
-            * 1e6
-            / 8760  # *n.snapshot_weightings.objective[0] #TODO change the way pset is sampled here
+    
+            #/ 8760  # *n.snapshot_weightings.objective[0] #TODO change the way pset is sampled here

```

Comment:
we need to fix this, either divide by 8760 or `n.snapshot_weightings.objective[0]`, but doing neither leads to a bug